tensorflow/docs_src/guide/datasets.md

```diff
@@ -356,6 +356,8 @@ with tf.Session() as sess:
   saver.restore(sess, path_to_checkpoint)
 ```
 
+Note that it is not allowed to save the state of the iterator which uses `shuffle`.
```

Comment:
Is this still true after 8666eff2359ccacd528dfda404a1f8ae35762542?

/cc @saxenasaurabh